sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/epkversion/ServiceItemLeaseV1.java

```diff
@@ -150,17 +149,11 @@ public ChangeFeedState getContinuationState(String containerRid, ChangeFeedMode
 
         //  Lease token are stored in Base64 encoded json - and contains the complete ChangeFeedState
         ChangeFeedState changeFeedState = ChangeFeedStateV1.fromString(this.continuationToken);
-        //  Calculating this token from epk based lease format
-        //  This token is then used to pass as lsn in form of etag.
-        String token = changeFeedState.getContinuation().getCurrentContinuationToken().getToken();
-        //  This token has extra quotes
-        token = token.replace("\"", "");
-
         return new ChangeFeedStateV1(
             containerRid,
             this.feedRangeInternal,
             changeFeedMode,
-            ChangeFeedStartFromInternal.createFromETagAndFeedRange(token, this.feedRangeInternal),
+            changeFeedState.getStartFromSettings(),
```

Comment:
why do we need to change this?

Reply:
I think the startFromSettings inside the ChangeFeedState should always reflect the original start from and we are depending on it to always send the start time.

Currently, if we resume from the existing lease or split happens, then we lost the original startFrom info, which I think is a bug

Reply:
i think the startFrom should always track the original startFrom settings of the changeFeedState and it is critical for us to keep sending the startFrom header properly for the following request. 

Currently, without this change, the original startFrom information will be missed when we resume processing from current lease or when a split happens